core/src/test/java/org/frankframework/parameters/DateParameterTest.java

```diff
@@ -417,4 +424,51 @@ public void testParameterFromDateToDate() throws Exception {
 		assertInstanceOf(Date.class, result);
 		assertEquals(date, result);
 	}
+
+	private void testDateParameterToString(DateFormatType type, String pattern, String input, String expectedOutput) throws ParseException, ConfigurationException {
+		DateFormat df = new SimpleDateFormat(pattern);
+		Date date = df.parse(input);
```

Comment:
Een beetje stom dat dit nu zo gebeurd maar ik snap waarom je het doet.
Ik vind wel dat de tests in de ParameterValue class thuishoren. Dan is het duidelijk wat we testen :)